core/utils/ip.h

```diff
@@ -67,6 +67,8 @@ struct Ipv4Prefix {
 
   // Returns true if ip is within the range of Ipv4Prefix
   bool Match(const be32_t &ip) const { return (addr & mask) == (ip & mask); }
+  // Returns the prefix length
+  uint32_t prefixLength() const {return ((32 - __builtin_clz(mask.raw_value())));}
```

Comment:
Perhaps rename the function to `prefix_length` (accessors may be named like variables as per Google C++ Style Guide)